roles/confluent.common/defaults/main.yml

```diff
@@ -4,6 +4,7 @@ zookeeper:
 broker:
   config:
     port: 9092
+    host: 0.0.0.0
```

Comment:
To keep this backwards compatible, should it be an empty string?

Reply:
Thank you for your comment. Yes, you are right. The effects of 0.0.0.0 and empty string are the same in runtime - kafka broker binds on all interfaces available:
```
[root@centos1 vagrant]# netstat -lnetp
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       User       Inode      PID/Program name    
tcp        0      0 0.0.0.0:111             0.0.0.0:*               LISTEN      0          13209      1/systemd           
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      0          17508      939/sshd            
tcp        0      0 127.0.0.1:25            0.0.0.0:*               LISTEN      0          18203      1527/master         
tcp6       0      0 :::111                  :::*                    LISTEN      0          13208      1/systemd           
tcp6       0      0 192.168.1.2:3888        :::*                    LISTEN      992        57840      17900/java          
tcp6       0      0 :::44470                :::*                    LISTEN      992        57664      17900/java          
tcp6       0      0 :::22                   :::*                    LISTEN      0          17553      939/sshd            
tcp6       0      0 ::1:25                  :::*                    LISTEN      0          18204      1527/master         
tcp6       0      0 :::36640                :::*                    LISTEN      992        60677      18815/java          
tcp6       0      0 :::9092                 :::*                    LISTEN      992        61681      18815/java          
tcp6       0      0 :::2181                 :::*                    LISTEN      992        57108      17900/java    
```

However, for the clean solution, it would be better to have the default set as empty string.
I have re-tested the provisioning using empty string and it works the same as before my changes.

I have made another commit with empty string as the default.